core/src/main/java/edu/wpi/grip/core/sources/CameraSource.java

```diff
@@ -165,6 +169,25 @@ public String getName() {
     }
 
     @Override
+    protected boolean updateOutputSockets() {
```

Comment:
> If you have multiple camera sources then that will throttle the pipeline to the slowest one.

I think this is what most people would want.  If you have two cameras at 24fps and 23fps, you wouldn't want to run at 47fps.